docs/tutorials/jupyter-sdk.rst

```diff
@@ -19,172 +25,117 @@ Jupyter can then be run using simply
 
     jupyter notebook
 
-Jupyter Notebooks and the Orquestra Workflow SDK must be installed in the same environment.
+Jupyter Notebooks and the Orquestra Workflow SDK must be installed in the same virtual environment.
```

Comment:
Suggestion: I'd either use "virtual environment" or "environment" throughout for clarity

Reply:
Good catch 🤦 a39cdf893d014cf2eb7d8aff349e63b4577cd8ff